cardinal/ecs/persona.go

```diff
@@ -193,3 +195,112 @@ func (w *World) GetSignerForPersonaTag(personaTag string, tick uint64) (addr str
 	}
 	return addr, nil
 }
+
+// TODO private component function used to temporarily remove circular dependency until we replace components.
```

Comment:
I created a ticket to capture the plugin work: https://linear.app/arguslabs/issue/WORLD-423/ecs-plugin-feature -- please add a link to the ticket in your comment here.